templates/zerver/help/send-a-message-in-a-different-language.md

```diff
@@ -0,0 +1,78 @@
+# Send a message in a different language
+
+To send a message in a different language you have to change your keyboard settings
+through your Operating System.
+
+# Change keyboard language on Linux
+
+{go-to-the.md} *System Settings* via the distibution specific application
+launcher.
+
+{go-to-the.md} *Text Entry* page.
+
+3. Add an input language by clicking on the plus
+(<i class="icon-vector-plus"></i>) icon.
+
+4. Select the language you want from the menu that appears.
+
+5. Next, select this language from the icon in your menu bar
+that is currently the abreviation for your current language. 
+This sets your current keyboard language to the selected language.
+
+6. If you would like to revert back to your previous language, reselect
+your previous language from the language selector in the menu bar.
+
+# Change keyboard language on Windows
+
+1. Click Start, type intl.cpl in the *Start Search* box, and then press
+ENTER.
+
+2. On the *Keyboards and Language* tab, click *Change keyboards*.
```

Comment:
I think single asterisks will make the text italic. I think that buttons and tabs should be in bold.